plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/StmtConversionVisitor.kt

```diff
@@ -138,20 +140,21 @@ object StmtConversionVisitor : FirVisitor<ExpEmbedding, StmtConversionContext<Re
         val right = data.convert(equalityOperatorCall.arguments[1])
 
         return when (equalityOperatorCall.operation) {
-            FirOperation.EQ -> convertEqCmp(left, right)
-            FirOperation.NOT_EQ -> Not(convertEqCmp(left, right))
+            FirOperation.EQ -> convertEqCmp(left, right, equalityOperatorCall.source)
+            FirOperation.NOT_EQ -> Not(convertEqCmp(left, right, equalityOperatorCall.source), equalityOperatorCall.source)
             else -> handleUnimplementedElement("Equality comparison operation ${equalityOperatorCall.operation} not yet implemented.", data)
         }
     }
 
-    private fun convertEqCmp(left: ExpEmbedding, right: ExpEmbedding): ExpEmbedding {
+    private fun convertEqCmp(left: ExpEmbedding, right: ExpEmbedding, pos: KtSourceElement?): ExpEmbedding {
```

Comment:
nit: looking at the rest of the code it seems like `pos` typically refers to a position, with `source` being used for `KtSourceElement`s.